app/assets/stylesheets/application.scss

```diff
@@ -125,3 +129,7 @@ html, body, .landing {
     color: white;
 }
 
+#follower_reason {
```

Comment:
Avoid using id selectors<br>Selector `follower_reason` should be written in lowercase with hyphens